src/app/system/audit-trails/view-audit/view-audit.component.html

```diff
@@ -0,0 +1,108 @@
+<div class="container" fxLayout="row" fxLayoutGap="2%" fxLayout.lt-md="column">
+
+  <div fxFlex="44%">
+
+    <mat-card>
+
+      <mat-card-content>
+
+        <div fxLayout="row wrap" class="content">
+
+          <div fxFlex="50%" class="mat-body-strong">
+            ID:
+          </div>
+
+          <div fxFlex="50%">
+            {{ auditTrailData.id }}
+          </div>
+
+          <div fxFlex="50%" class="mat-body-strong">
+            Status:
+          </div>
+
+          <div fxFlex="50%">
+            {{ auditTrailData.processingResult }}
+          </div>
+
+          <div fxFlex="50%" class="mat-body-strong">
+            User:
+          </div>
+
+          <div fxFlex="50%">
+            {{ auditTrailData.maker }}
+          </div>
+
+          <div fxFlex="50%" class="mat-body-strong">
+            Action:
+          </div>
+
+          <div fxFlex="50%">
+            {{ auditTrailData.actionName }}
+          </div>
+
+          <div fxFlex="50%" class="mat-body-strong">
+            Entity:
+          </div>
+
+          <div fxFlex="50%">
+            {{ auditTrailData.entityName }}
+          </div>
+
+          <div fxFlex="50%" class="mat-body-strong">
+            Resource ID:
+          </div>
+
+          <div fxFlex="50%">
+            {{ auditTrailData.resourceId }}
+          </div>
+
+          <div fxFlex="50%" class="mat-body-strong">
+            Date:
+          </div>
+
+          <div fxFlex="50%">
+            {{ auditTrailData.madeOnDate }}
+          </div>
+
+          <div fxFlex="50%" class="mat-body-strong">
+            Office:
+          </div>
+
+          <div fxFlex="50%">
+            {{ auditTrailData.officeName }}
+          </div>
+
+        </div>
+
+      </mat-card-content>
+
+    </mat-card>
+
+  </div>
+
+  <br>
+
+  <br>
+
+  <div class="mat-elevation-z8" fxFlex>
```

Comment:
Its still covering extra height.

Reply:
Fixed. Please refer to the screenshot. Reopen it if you feel this is still unsolved.

Reply:
It is still covering extra height.

Reply:
@Chirag this is left.

Reply:
@abhaychawla Resolved now. This existed because we added `space-between` which asked for a fixed space in between the two cards and we had a fixed size defined for cards so first card took all its space required but then second card could only acquire the remaining space.